src/sarif_converter.ts

```diff
@@ -50,8 +50,9 @@ const getTools = (analysisResults: IAnalysisResult, suggestions: ISarifSuggestio
       2: 'warning',
       3: 'error',
     }[suggestion.severity];
-
-    const suggestionId = suggestion.id;
+    // payload comes as URIencoded
+    const language = suggestion.id.split('%2F')[0];
+    const suggestionId = `${language}/${suggestion.rule}`;
```

Comment:
would rename `suggestionId`  to `ruleId` as otherwise it's quite confusing, as there are multiple suggestions.